alpine-rails-nginx/Dockerfile

```diff
@@ -19,24 +21,20 @@ RUN adduser -DSH nginx \
 
 # Create system directories and service symlinks
 RUN setup-directories.sh  root  r  /etc/service/nginx /etc/service/rails \
-  && setup-directories.sh nginx r  /etc/nginx \
   && setup-directories.sh nginx rw /var/run/nginx /var/cache/nginx /var/log/nginx \
-  && setup-directories.sh rails rw /opt/rails /var/run/rails \
-  && ln -s -- /etc/sv/nginx /etc/service/nginx/run \
-  && ln -s -- /etc/sv/rails /etc/service/rails/run
+  && setup-directories.sh rails r  /opt/rails \
+  && setup-directories.sh rails rw /var/run/rails /opt/rails/log /opt/rails/tmp \
+  && ln -s /etc/sv/nginx /etc/service/nginx/run \
+  && ln -s /etc/sv/rails /etc/service/rails/run \
+  && chown rails: /usr/bin/bundle
```

Comment:
This is because of the tight umask. The file is created as owned by root, and only executable by root.

I *could* change it to be executable by "other", or add an acl entry to allow it to be executed if you think that's cleaner?